web/cypress/integration/fba-map-page.spec.ts

```diff
@@ -17,4 +13,15 @@ describe('Fire Behaviour Advisory Page', () => {
     cy.getByTestId('fire-center-dropdown').should('be.visible')
     cy.getByTestId('fba-map').should('be.visible')
   })
+
+  it('Sets the fireCenter in local storage when it is changed in dropdown', () => {
+    cy.wait('@getStations')
+    cy.getByTestId('fire-center-dropdown')
+      .click()
+      .type('{downArrow}')
+      .type('{enter}')
+      .should(() => {
+        expect(localStorage.getItem('preferredFireCenter')).to.equal('50')
```

Comment:
Did not know cypress has access to `localStorage` right off the bat like this. Cool! Does this test fail if you change the equal assertion to another value?

Reply:
Yes because right now it's expecting the local storage to be the id of the first item in the dropdown when selected.

Reply:
Phew - I was worried this was going to be an issue. Have tripped up on that before.